### PR TITLE
Jenkins improve

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Description 
- bump karma version (fix socket hanging)
- ember test will use existing build

/cc @Wikia/x-wing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "start": "nodemon --delay 2",
     "start-on-change": "nodemon --delay 2 --on-change-only",
     "test": "npm run build-dev && npm run test-front && npm run test-server",
-    "test-front": "gulp test-common && gulp test-auth && cd front/main && ember test",
+    "test-front": "gulp test-common && gulp test-auth && cd front/main && ember test --path ../../www/front/main",
```
Comment (hakubo) [inline package.json]: What if I haven't build main before?

**Comment by rogatty** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (rogatty) [inline package.json]: It won't work anyway, notice that `npm run test` does the build first and then runs `test-front`.

Currently `test-front` isn't independent.

Comment (hakubo) [inline package.json]: gotcha

**Comment by hakubo** [comment]
lgtm

**Pull request merged**